src/Blockcore/Controllers/ConnectionManagerController.cs

```diff
@@ -7,14 +7,17 @@
 using Blockcore.Utilities;
 using Blockcore.Utilities.Extensions;
 using Blockcore.Utilities.JsonErrors;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using Swashbuckle.AspNetCore.Annotations;
 
 namespace Blockcore.Controllers
 {
     /// <summary>
     /// A <see cref="FeatureController"/> that implements API and RPC methods for the connection manager.
     /// </summary>
+    [Authorize]
```

Comment:
Seems correct, will this also be required for localhost?

Reply:
There won't be any difference where traffic comes from.